ios/Delegates/ApiClientSessionDelegate.swift

```diff
@@ -15,25 +15,31 @@ class ApiClientSessionDelegate: SessionDelegate {
                 if session.trustSelfSignedServerCertificate {
                     credential = URLCredential(trust: challenge.protectionSpace.serverTrust!)
                     disposition = .useCredential
+                    completionHandler(disposition, credential)
```

Comment:
These changes are under  `if authMethod == NSURLAuthenticationMethodServerTrust` and ` if session.trustSelfSignedServerCertificate`. Shouldn't we also check here for the serverTrust in case of SSL pinning?

Reply:
so the authentication for server trust is to check for the SSL certificate indeed, if the library is configured to allow a self-sign cert (which we don't really use at MM) then I did not see the point of also checking for the SSL Pinning

Reply:
I meant similar to what we do here: https://github.com/mattermost/react-native-network-client/pull/129/files#diff-312a94b8092b785515c7927ca031010b7dd22f9a23f50bf26cea11e82f415871R106-R111

```
 if session.trustSelfSignedServerCertificate {
  //do what we currently do
else
//check for SSL pinning
```

Reply:
Oh no need, that is happening with the call to super and handled in AlamoFire